<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="52943" started-at="2018-04-27T11:32:25Z" finished-at="2018-04-27T11:33:18Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="52943" started-at="2018-04-27T11:32:25Z" finished-at="2018-04-27T11:33:18Z">
      <class name="com.tests.LoginTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.tests.LoginTest@6e1ec318]" name="setup" is-config="true" duration-ms="26869" started-at="2018-04-27T17:02:25Z" finished-at="2018-04-27T17:02:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="freeCrmTitleTest()[pri:1, instance:com.tests.LoginTest@6e1ec318]" name="freeCrmTitleTest" duration-ms="80" started-at="2018-04-27T17:02:52Z" finished-at="2018-04-27T17:02:52Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [#1 Free CRM for Any Business: Online Customer Relationship Software] but found [Free CRM software in the cloud powers sales and customer service]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [#1 Free CRM for Any Business: Online Customer Relationship Software] but found [Free CRM software in the cloud powers sales and customer service]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.tests.LoginTest.freeCrmTitleTest(LoginTest.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- freeCrmTitleTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.tests.LoginTest@6e1ec318]" name="tearDown" is-config="true" duration-ms="780" started-at="2018-04-27T17:02:52Z" finished-at="2018-04-27T17:02:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.tests.LoginTest@6e1ec318]" name="setup" is-config="true" duration-ms="24292" started-at="2018-04-27T17:02:53Z" finished-at="2018-04-27T17:03:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="freemCRMLogoTest()[pri:2, instance:com.tests.LoginTest@6e1ec318]" name="freemCRMLogoTest" duration-ms="113" started-at="2018-04-27T17:03:17Z" finished-at="2018-04-27T17:03:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- freemCRMLogoTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.tests.LoginTest@6e1ec318]" name="tearDown" is-config="true" duration-ms="747" started-at="2018-04-27T17:03:17Z" finished-at="2018-04-27T17:03:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.tests.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
